### ISSUE TITLE
bug(conda): files written to the site-packages directory are included in the output package

### ISSUE THREAD

**Issue by cfobel** [sci-bots/nadamq#4]
# Problem

Files written to the `site-packages` directory are **not included** in the output package.

The following error is raised when trying to copy from `%SP_DIR%`, i.e., the `site-packages` directory:

```sh
Finished generating code
copying build\lib.win-amd64-3.6\nadamq\NadaMq.cp36-win_amd64.pyd -> nadamq
Test auto-buffer based on data content. ... ok
Test setting data while allocating larger buffer. ... ok
Test setting allocating buffer without setting data contents. ... ok
Test no buffer allocation. ... ok
Test default packet allocation, i.e., no iuid, type, or data/buffer. ... ok
Test serialization of ``cPacket`` containing ``"hello, world!"``. ... ok
test_packet_parser.test_parse_data ... ok
test_packet_parser.test_parse_ack ... ok
test_packet_parser.test_parse_nack ... ok
.. versionadded:: 0.13 ... ok
.. versionadded:: 0.13 ... ok

----------------------------------------------------------------------
Ran 11 tests in 0.428s

OK
c:\dbot\nadamq_1519938130386\_h_env\Lib\site-packages
The system cannot find the path specified.
Traceback (most recent call last):
  File "C:\Users\chris\Miniconda2-x64\Scripts\conda-build-script.py", line 5, in <module>
    sys.exit(conda_build.cli.main_build.main())
  File "C:\Users\chris\Miniconda2-x64\lib\site-packages\conda_build\cli\main_build.py", line 413, in main
    execute(sys.argv[1:])
  File "C:\Users\chris\Miniconda2-x64\lib\site-packages\conda_build\cli\main_build.py", line 404, in execute
    verify=args.verify)
  File "C:\Users\chris\Miniconda2-x64\lib\site-packages\conda_build\api.py", line 193, in build
    need_source_download=need_source_download, config=config, variants=variants)
  File "C:\Users\chris\Miniconda2-x64\lib\site-packages\conda_build\build.py", line 1962, in build_tree
    notest=notest,
  File "C:\Users\chris\Miniconda2-x64\lib\site-packages\conda_build\build.py", line 1217, in build
    windows.build(m, build_file)
  File "C:\Users\chris\Miniconda2-x64\lib\site-packages\conda_build\windows.py", line 273, in build
    check_call_env(cmd, cwd=src_dir)
  File "C:\Users\chris\Miniconda2-x64\lib\site-packages\conda_build\utils.py", line 678, in check_call_env
    return _func_defaulting_env_to_os_environ(subprocess.check_call, *popenargs, **kwargs)
  File "C:\Users\chris\Miniconda2-x64\lib\site-packages\conda_build\utils.py", line 674, in _func_defaulting_env_to_os_environ
    return func(_args, **kwargs)
  File "C:\Users\chris\Miniconda2-x64\lib\subprocess.py", line 186, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['cmd.exe', '/c', 'bld.bat']' returned non-zero exit status 1
```

# Cause 

This issue is caused by `conda-build>=3.0`:

> **TL;DR**: **If you use the new `{{ compiler() }}` jinja2** ... **you also must move anything that is not strictly a build tool into your host dependencies. This includes python, python libraries**, and any shared
libraries that you need to link against in your build. Examples of build tools
include any `{{ compiler() }}`, make, autoconf, perl (for running scripts, not
installing perl software), python (for running scripts, not for installing
software).

[1]: https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#host
